wp-content/themes/core/assets/css/src/admin/block-editor/blocks/tribe.pcss

```diff
@@ -0,0 +1,13 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Modern Tribe Namespaced Block Styles
```

Comment:
@faction23 , I've moved the Gutenberg partials to the `block-editor` editor directory for consistency.

Reply:
noted, good call